src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+
+```
+SELECT uniques(pageUrl) from PageView LIMIT MAX 
+```
+
+<Callout variant="tip">
+New to NRQL? Check out our [introduction to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to learn about how to use our custom query language to help maximize your data!
+</Callout>
+
+    </Step>
+    <Step>
+### Validate browser URL grouping
+
+After you've made sure that your instrumented pages report their data correctly, it's time to ensure browser segments are captured correctly. If you're unfamiliar with segments, they're simply the text between two `/`s in a URL or the `.`s of a domain name. For example, in the URL `website.com/product/widget-name`, the segments are:
+* `website`
+* `.com`
+* `product`
+* `widget-name`
+
+When you have a lot of URLs with a lot of segments, you can abridge them so that `website.com/product/widget-name` becomes `website.com/` **or** `website.com/product/`. In this example, the first abridged URL isn't particularly useful, but the second one is a useful way of grouping customer experience data for the product.
+
+<Callout variant="tip">
+Not sure whether you need to tune your configuration? Import the [Segment allow-list investigation dashboard](https://newrelic.com/instant-observability/browser-segment-investigation/eb24e234-90aa-4908-972d-64d6d56b557e) to help.
+</Callout>
+
+<img
+  title="Segment allow-list (aka white-list) investigation"
+  alt="Segment allow-list (aka white-list) investigation"
+  src={cxSegmentWhitelistInvestigation}
+/>
+
+Once you've identified all your segments, use the [Segment allow-lists in browser](/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls/#adding) to add them.        
+    </Step>
+    <Step>
+### Understand how you will segment your data
+
+Now that you've validated your browser URL grouping, the next step is to make your customer experience data easier to understand by breaking it out into different segments. Unlike the previous step, segments here don't refer to sections of URLs (as in the [segment allow lists](https://docs.newrelic.com/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls/#adding)), but instead refer to groups of data.
+
+Often, segments can differentiate based on various factors. For example:
+* Most of the users in the US, Canada, and EMEA experience 2 seconds or better to first input delay, while users in Malaysia and Indonesia experience 4 seconds.
+* Customers buying car insurance typically see 1 second to largest contentful paint. For home insurance, it's 4 seconds.
+* In one week, there can be 700,000 page views on mobile browser apps compared to 300,000 on desktop.
+
+Typical segmentation involves breaking down user experience into categories like those above to help find weak points in the overall experience. You usually group segments by the following categories:
+
+<CollapserGroup>
```

Comment:
The content in this collapser is the good stuff. Very useful info. Almost seems a shame to hide it in collapsers

Reply:
Thanks! I wish we didn't have to rely on them either, but these docs are so variable I think we need to let customers pick and choose what they want to see to avoid potential bloat.

Reply:
Thanks for the extra context, works for me